components/atoms/ProfileImage/ProfileImage.js

```diff
@@ -1,17 +1,33 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const StyledImage = styled.img`
+const StyledImageWrapper = styled.div`
+  position: relative;
   display: inline-block;
   width: 1.875rem;
   height: 1.875rem;
   font-size: 0;
   border-radius: 50%;
+  overflow: hidden;
+`;
+const StyledImage = styled.img`
```

Comment:
I propose to compare ways how you can display an image. I added a dark background to the container so that you can see how the image stretches inside it:

Using absolute positions:
![Screen Shot 2020-10-06 at 13 14 09](https://user-images.githubusercontent.com/53777036/95189142-229c5a80-07d6-11eb-8fdb-f28a2411c924.png)

Using object-fit property:
![Screen Shot 2020-10-06 at 13 14 24](https://user-images.githubusercontent.com/53777036/95189188-32b43a00-07d6-11eb-9a06-372039338347.png)

How do you look at using the 2nd way? In my opinion, it gives the better result.

Reply:
Agree with you @Nikita-Polyakov 
But `object-fit` property is not supported by IE11. That's why I used a cross-browser solution.
We can decide, that we don't support IE in this repo and use `object-fit`. That do you think?

Reply:
I think we should support ie11. 
I suggest another solution with the same result as with `object-fit:cover`:

```
{
   height: 100%;
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
}
```